armi/materials/uraniumOxide.py

```diff
@@ -55,7 +55,7 @@ class UraniumOxide(material.FuelMaterial, material.SimpleSolid):
     propertyUnits = {"heat capacity": "J/mol-K"}
 
     propertyValidTemperature = {
-        "density": ((300, 3100), "K"),
+        "density3": ((300, 3100), "K"),
```

Comment:
Is this the only material you found that needed this update?

Reply:
Well, I randomly happened across this one. 

Perhaps we could look through all the materials for similar mis-namings.